src/views/presentations.hbs

```diff
@@ -14,20 +14,11 @@ Searching...
        hx-trigger="input changed delay:500ms, search" 
        hx-post="/search" 
        hx-target="#search-results" 
-       hx-swap="outerHTML"
```

Comment:
Now that `#search-results` is a static wrapper around the presentation rows, we can use htmx default swap behavior of `innerHTML`